.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\C:\\Users\\Dsvk2\\Source\\Repos\\opensource.guide",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
nit: if we keep, can we ensure newline at end of file please 🙏

Reply:
Can I remove this string from github???

Reply:
Thank you for report. I didn't see that I uploaded the .vs folder. I've already solved it, so you can proceed with the review.